portfolio/src/main/webapp/script.js

```diff
@@ -32,3 +62,17 @@ function addRandomFact() {
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = fact;
 }
+
+async function heyBilly() {
+  const response = await fetch('/data');
+  const quote    = await response.text();
+  document.getElementById('hey-container').innerHTML = quote;
+}
+
+// Not in use, just for practice and reference
+function heyBillyArrow() {
+  fetch('/data')     .then(response => {
```

Comment:
Remove inner whitespace.

Reply:
In general this is a little difficult to decipher. I recommend organizing like this:

```
fetch('/data')
  .then(response => {
    ...
  })
  .then(quote => {
    ...
  });
```